Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:10
+MAINTAINER devops@optimumenergyco.com
```

Comment:
Last time I used `MAINTAINER` I had some linting come up within vscode that this dockerfile field is [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated). I think we will want to change this to `LABEL maintainer="devops@optimumenergyco.com"`

Reply:
Good call. I think we can get rid of this entirely. These are private images, and there's no benefit to adding stuff like this. It's like adding a copyright notice to code.